src/Core/Framework/Api/ApiDefinition/Generator/Schema/StoreApi/components/parameters/criteria.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
is it by design that this is a "plain" json file? All the other component definitions start with 
```
    "openapi": "3.1.0",
    "info": [],
    "paths": [],
```

Is that not needed anymore? Because as far as I see this file is loading in the same way as the others 🤔

Reply:
ah i see 👍 

No IMHO let's keep it simple, no need to overcomplicate